src/styles/blocks/variables.scss

```diff
@@ -0,0 +1,20 @@
+$text-size: 30px;
+$day-size: 100px;
+$gap-size: 1px;
+$week-length: 7;
+$gap-counts: $week-length - 1;
+$calendar-width: $day-size * $week-length + $gap-size * $gap-counts;
+$days: (
+  "mon": 0,
+  "tue": ($day-size + $gap-size),
```

Comment:
Better to store here only multipliers for day number and move this calculation to function